engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaSinkFactory.scala

```diff
@@ -9,14 +9,14 @@ import pl.touk.nussknacker.engine.api.editor.{DualEditor, DualEditorMode, Simple
 import pl.touk.nussknacker.engine.api.process.{ProcessObjectDependencies, Sink, SinkFactory}
 import pl.touk.nussknacker.engine.api.{MetaData, MethodToInvoke, ParamName}
 import pl.touk.nussknacker.engine.flink.api.process.BasicFlinkSink
-import pl.touk.nussknacker.engine.kafka.serialization.{FixedSerializationSchemaFactory, SerializationSchemaFactory}
 import pl.touk.nussknacker.engine.kafka.BaseKafkaSinkFactory._
+import pl.touk.nussknacker.engine.kafka.serialization.{FixedKafkaSerializationSchemaFactory, KafkaSerializationSchemaFactory}
 
-class KafkaSinkFactory(serializationSchemaFactory: SerializationSchemaFactory[Any], processObjectDependencies: ProcessObjectDependencies)
-  extends BaseKafkaSinkFactory(serializationSchemaFactory, processObjectDependencies) {
+class KafkaSinkFactory(schemaSerializerFactory: KafkaSerializationSchemaFactory[Any], processObjectDependencies: ProcessObjectDependencies)
```

Comment:
`serializationSchemaFactory` was better IMO. We didn't introduced new `schemaSerializer` phrase

Reply:
Right